apis/v1alpha2/gateway_types.go

```diff
@@ -231,8 +270,8 @@ const (
 	// Deprecated: use the "Accepted" condition with reason "Accepted" instead.
 	GatewayReasonScheduled GatewayConditionReason = "Scheduled"
 
-	// This reason is used with the "Accepted" condition when no controller has
-	// reconciled the Gateway.
+	// This reason is used with the "Accepted", "Programmed" and "Ready"
+	// conditions when no controller has reconciled the Gateway.
 	GatewayReasonPending GatewayConditionReason = "Pending"
```

Comment:
I think we may have gotten this wrong in the GEP. It seems like the current recommendation in API Conventions is "True/False/Unknown", where "Unknown" is used similarly to how we're recommending "Pending" being used: https://github.com/kubernetes/community/blob/master/contributors/devel/sig-architecture/api-conventions.md#typical-status-properties.

/cc @youngnick

Reply:
It could use clarification, but I thought that this was for the defaults, so the Type should be `Accepted`, `Programmed`, or `Ready`, the status is `Unknown`, and then the Reason is `Pending`.

Maybe like this?
```suggestion
	// This reason is used with the "Accepted", "Programmed" and "Ready"
	// conditions when the status is "Unknown" and no controller has reconciled the Gateway.
	GatewayReasonPending GatewayConditionReason = "Pending"
```